include/aikido/planner/ConfigurationToConfigurations.hpp

```diff
@@ -47,14 +47,14 @@ class ConfigurationToConfigurations : public Problem
   std::size_t getNumGoalStates() const;
 
   /// Returns goal states.
-  const GoalStates& getGoalStates() const;
+  const GoalStates getGoalStates() const;
```

Comment:
Since we are returning a copy, I do not think we need to return `const GoalStates` but if others think the intention is clear from `const` we can let it be.